pkg/manifestgen/manifestgen.go

```diff
@@ -89,6 +94,7 @@ func New(reporegistry *reporegistry.RepoRegistry, opts *Options) (*Generator, er
 		rpmDownloader:     opts.RpmDownloader,
 		sbomWriter:        opts.SBOMWriter,
 		customSeed:        opts.CustomSeed,
+		overrideRepos:     opts.OverrideRepos,
```

Comment:
Nitpick: would it make sense to return an `error` in case `opts.OverrideRepos` and `reporegistry` are both provided?, Because in such case, the `reporegistry` will always be ignored. Similarly, would an error make sense if none is provided?